.github/workflows/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Build Matlab bindings'
+description: 'Installs precice and builds the preCICE matlab bindings in the directory. Ends in `$GITHUB_WORKSPACE/matlab-bindings`'
```

Comment:
preCICE* and either Matlab* or MATLAB* everywhere (please check the correct nomenclature)